browser-test/src/support/admin_questions.ts

```diff
@@ -305,37 +315,19 @@ export class AdminQuestions {
     await this.expectDraftQuestionExist(questionName, questionText);
   }
 
-  async addRepeaterQuestion(questionName: string,
-    description = 'repeater description',
-    questionText = 'repeater question text',
-    helpText = 'repeater question help text') {
+  async addEnumeratorQuestion(questionName: string,
+    description = 'enumerator description',
+    questionText = 'enumerator question text',
+    helpText = 'enumerator question help text',
+    enumeratorName = 'does not repeat') {

```

Comment:
question: why does this default to 'does not repeat'? Does that mean the other questions that are by default added to the 'does not repeat' enumerator will instead be added to this one if it exists?

Reply:
Never mind, I understand now! This enumerator by default does not repeat - maybe we could add a comment to explain this one?

Reply:
I can add a small comment here, but these arguments match all the other `addXQuestion` methods and I don't think they all need a small comment.

Reply:
No worries! Sounds good